sdk/tests/deqp/functional/gles3/es3fSyncTests.js

```diff
@@ -170,11 +179,15 @@ goog.scope(function() {
             gl.flush();
         if (this.m_caseOptions & es3fSyncTests.CaseOptions.FINISH_BEFORE_WAIT)
             gl.finish();
+        this.m_state = es3fSyncTests.State.VERIFY;
+    };
 
-        // Wait for sync object
 
+    es3fSyncTests.FenceSyncCase.prototype.verify = function() {
         /** @type {number} */ var waitValue = 0;
```

Comment:
Can we clean up the bufferedLogToConsole in this function a little bit?  There could be overwhelmingly large number of them in the console and in the webgl log.